### ISSUE TITLE
Node.js 16.x is the current Long Term Support (LTS)

### ISSUE THREAD

**Issue by fabiel-leon** [aws/aws-lambda-base-images#33]
https://nodejs.org/en/ 

implement this in a new base image please :pray:

**Comment by alexbaileyuk** [comment]
Duplicate of https://github.com/aws/aws-lambda-base-images/issues/14

**Comment by y4my4my4m** [comment]
@carlzogh @krk 
Sorry to ping you guys directly, but is there any information on this?
I've searched and searched and it seems that there is zero information out there for developers.
AWS is too big to be leaving developers in the dark like this.

Thank you

**Comment by rawpixel-vincent** [comment]
> AWS is too big to be leaving developers in the dark like this.

AWS is big enough to leave developers in the dark like this.

**Comment by dl748** [comment]
Placeholder for closed issue #14

**Comment by DavidTanner** [comment]
https://github.com/aws/aws-lambda-base-images/issues/14#issuecomment-1108679026

```
Thank you everyone for the continued engagement and feedback on this issue. The AWS Lambda Node 16 runtime implementation is in progress, and we are currently on track to launch in mid-May. Our Node 16 runtime will continue to use AWS SDKv2, and we plan to transition to AWS SDKv3 with Node 18.
```
